setup.cfg

```diff
@@ -1,6 +1,56 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = odltools
+summary = Tools for troubleshooting OpenDaylight
+description-file =
+    README.rst
+author = Sam Hague
+author-email = shague@gmail.com
+home-page = http://github.com/shague/odltools
+license =  Apache-2
+classifier =
+        Development Status :: 1 - Planning
+        Environment :: Console
+        Environment :: OpenStack
+        Intended Audience :: Developers
+        Intended Audience :: Information Technology
+        Intended Audience :: System Administrators
+        License :: OSI Approved :: Apache Software License
+        Natural Language :: English
+        Operating System :: OS Independent
+        Programming Language :: Python
+        Programming Language :: Python :: 2
+        Programming Language :: Python :: 2.7
+        Programming Language :: Python :: 3
+        Programming Language :: Python :: 3.3
+        Programming Language :: Python :: 3.4
+        Programming Language :: Python :: 3.5
+        Programming Language :: Python :: 3.6
+        Topic :: Software Development
+        Topic :: System :: Networking
+        Topic :: System :: Systems Administration
+        Topic :: Utilities
 
-[egg_info]
-#tag_build = dev
-tag_build =
+[global]
+setup-hooks =
+    pbr.hooks.setup_hook
+
+[entry_points]
+console_scripts =
+    odltools = odltools.__main__:main
+
+[files]
+packages =
+    odltools
```

Comment:
Does the skip including the tests?

Reply:
Yeah it automatically will not include tests dir.  But I think it may include it since you have recursive-include odltools/tests in the MANIFEST.in.  We could also remove the MANIFEST.in if we want to, pbr automatically generates it:
https://docs.openstack.org/pbr/latest/user/features.html#manifest